Penteract/Assets/Scripts/HUDController.cpp

```diff
@@ -266,13 +275,12 @@ void HUDController::ResetHealthRegenerationEffects(float currentHp) {
 	}
 }
 
-void HUDController::ResetCooldownProgressBar()
-{
+void HUDController::ResetCooldownProgressBar() {
 	if (!swapingSkillCanvas) {
 		return;
 	}
 
-	AbilityRefreshEffectProgressBar* pef = GET_SCRIPT(swapingSkillCanvas->GetChildren()[2], AbilityRefreshEffectProgressBar);
+	AbilityRefreshEffectProgressBar* pef = GET_SCRIPT(swapingSkillCanvas->GetChildren()[HIERARCHY_INDEX_SWAP_ABILITY_EFFECT], AbilityRefreshEffectProgressBar);
```

Comment:
I would do this in two steps because we are not checking the children's vector size as other methods do:

- First get the child
- Check the child is not null and then get the script 

OnHealthLost does exactly this proposal

Reply:
Done.